exasol_script_languages_developer_sandbox/runtime/ansible/roles/jupyter/tasks/motd.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: read current server config file
+  shell: "cat {{user_home}}/.jupyter/jupyter_server_config.json"
+  register: jupyter_server_config
+
+- name: save the Json data to a Variable as a Fact
+  set_fact:
+    jsondata: "{{ jupyter_server_config.stdout | from_json }}"
```

Comment:
These facts are global names, so name the fact accordingly

Reply:
Done